package.json

```diff
@@ -49,20 +49,23 @@
     "package.json": "npm run format:packagejson"
   },
   "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
```

Comment:
These also come installed in `backpack-react-scripts`, as they come in `react-scripts`. Our pattern in this repo is to have `eslint-config-skyscanner` able to work independently, so have installed them directly here.